### PR TITLE
Typography Changes-1

### PR DESCRIPTION
**Pull request by anmol5936**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [ ] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [ ] Make sure you are making a pull request against the **default branch** (left side). Also you should start *your branch* off *default branch*.
- [ ] Check the commit's or even all commits' message styles matches our requested structure.
- [ ] Check your code additions will fail neither code linting checks nor unit test.
- [ ] I have added necessary documentation (if appropriate)

### Description
Please describe your pull request.

❤️Thank you!

### Post merge checklist
- [ ] Follow steps from the [guidelines for contributing](../CONTRIBUTING.md) to this repository.
- [ ] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### CODE REVIEW

**Comment by Lucif3r-in** [comment]
@anmol5936 Keep a prop for italics

**Comment by Lucif3r-in** [comment]
Hey @anmol5936, Can you use the typography in the playground once so that we can check it out [playground](https://github.com/dscnitrourkela/project-oregano/blob/hacknitr5.0/src/pages/playground.js)

File: src/components/JoinUsSection/JoinUsSection.js
Comment (Lucif3r-in) [inline src/components/JoinUsSection/JoinUsSection.js:1]: This file is not related to this PR remove it

**Comment by anmol5936** [comment]
not required

**Comment by Lucif3r-in** [comment]
LGTM!🚀
@SirSimon162 Review once

**Comment by SirSimon162** [comment]
LGTM 🚀
